Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -176,6 +176,19 @@ public virtual void OnValidate()
             }
         }
 
+#if UNITY_EDITOR
+        void Reset()
+        {
+            // Prevent adding NetworkManager to parent or child of another NetworkManager
+            foreach (NetworkManager networkManager in transform.root.GetComponentsInChildren<NetworkManager>().ToList())
```

Comment:
- why .ToList()?
- why does it use transform.root? if we have two network managers, one of them a child of the other, GetComponentsInChildren would be enough or not?

Reply:
.toList because the collection is modified within the loop...duh.
transform.root because we don't know which is triggering Unity...the parent or the child.